src/components/ToDoContainer.jsx

```diff
@@ -31,19 +35,72 @@ export default function ToDoContainer() {
     localStorage.setItem('todos', JSON.stringify(todos));
     if (todos?.length) return;
   }, [todos]);
+  
+  const style1={
```

Comment:
Please use Tailwindcss only. To maintain the coding style of the application

Reply:
Sure